### DISCUSSION TITLE
Discord server

### DISCUSSION THREAD

**Discussion by vladmandic** [vladmandic/sdnext#1059] [Announcements]
Open invite at <https://discord.gg/VjvR2tabEX>
Second link: <https://discord.gg/sd-next-federal-batch-inspectors-1101998836328697867>

**Comment by vladmandic** [comment]
@Aptronymist thanks for pointing the wrong link

**Reply by vladmandic** [reply]
updated like to avoid discord expiration

**Comment by myndxero** [comment]
Ohhh more ways to poke vlad!

Sorry been mia so much, errr I guess more appropriate to apologize when I'm here as it's prob cuz of an issue or something haha. Hope things are going well with your fork, please don't burn out. I feel like the success you had here got auto1111 moving again but I do not wish to rely on his again. I remain your biggest fan!

BTW - I am losing my mind trying to remember the name of the person with their own UI that excelled at being pretty, you were gonna collab maybe? Did I miss it!? o.o

**Reply by DrJKL** [reply]
Anapnoe?

**Reply by vladmandic** [reply]
and yes, we're working on integrating :)

**Comment by myndxero** [comment]
Yes. Found the name short while ago. Kept searching for anapnea ... dug
into my browsing history to find it haha.

**Comment by PyroclasticFloe** [comment]
I'm getting discord telling me the link is invaild.

**Reply by vladmandic** [reply]
i've just updated the link as discord invites auto-expire after a week

**Reply by PyroclasticFloe** [reply]
Thanks for updating the link!

**Reply by Centurion-Rome** [reply]
![image](https://github.com/vladmandic/automatic/assets/75789136/35e01430-3210-4fc8-a449-4a4c94a3e62a)

**Reply by Centurion-Rome** [reply]
ok link is working again :)

**Comment by DoughyInTheMiddle** [comment]
If both are broken are we to just assume we got banned?

I know I DM'd @brknsoul without asking for permission first, but I've been on the Discord for a LONG time, and without any warning it seems I just got booted for ONE infraction.

**Reply by vladmandic** [reply]
your github account performed several malicious edits to wiki page pointing to a download of a zip file with virus.

**Reply by brknsoul** [reply]
I booted you because, as vlad said, multiple wiki edits pointing to infected zip;
![image](https://github.com/user-attachments/assets/835a8b15-a0a1-4829-b74a-01932e3237b0)

![image](https://github.com/user-attachments/assets/4ef0abc7-1a9c-4ede-908e-c969bfe01146)

**Reply by DoughyInTheMiddle** [reply]
Whaaat? I have zero clue how to do any of that!  Seriously, outside of some VBA coding that I haven't done in YEARS I have no clue how to do any of that. My coding skills are way way behind to even do the data analysis I did back in the day, much less running something malicious like that.

I'm not even sure how to get rid of that other than maybe a Symantec scan and Windows Defender scan.

Seriously, I'm so stunned I don't even know what to type.

**Reply by brknsoul** [reply]
We're not saying you created the virus, we're saying you shared a malicious link, multiple times.

**Reply by DoughyInTheMiddle** [reply]
> We're not saying you created the virus, we're saying you shared a malicious link, multiple times.

As in I would have edited the Wiki and tried to insert links to something malicious? 

I went to the repo for the first time (again, in months) on the 26th to download the software and saw that the Wiki had been greatly expanded/updated since the last time I'd used it. I wanted to verify where Python was, did so, cloned the repo.
![image](https://github.com/user-attachments/assets/7c67d605-3d0c-4988-b251-19a59e881d34)

Something was wonky, so I went back on the 27th and downloaded it again, and got it just fine.
![image](https://github.com/user-attachments/assets/0a4162a5-c189-485b-8977-080589dbd11c)

Then, I tried using it most of the day on the 27th, was having a helluva time getting any usable images, and that's when I started posting in the help chat on Discord.  

I just ran a Windows Defender scan and it found nothing. I'll run a Spybot scan too. 

Honestly, I have no clue how I could have made those edits. I don't even see where there's a place to select "edit" (it's not like it's clear like on Wikipedia or other software). 

Is there anything I can do to ensure this won't happen again AND get access to the Discord again?

**Reply by brknsoul** [reply]
Change your github and discord passwords (and any other similar passwords).

Respond to this comment when you've done so and I'll unban you.

**Reply by DoughyInTheMiddle** [reply]
> Change your github and discord passwords (and any other similar passwords).
> 
> Respond to this comment when you've done so and I'll unban you.

Done. Beefed both passwords up even stronger than than they already were before. 

Also, Spybot found nothing other than random low-priority cookies.

**Reply by brknsoul** [reply]
OK, You've been unbanned. Welcome back. ;)
